packages/dds/merge-tree/src/client.ts

```diff
@@ -1002,7 +1002,7 @@ export class Client {
         let clientId: number;
         if (op) {
             clientId = this.getOrAddShortClientId(op.clientId);
-            seq = op.sequenceNumber;
+            seq = op.referenceSequenceNumber;
```

Comment:
can you add a unit test for this in this package. just a single regression test should be enough. look at the file client.applyMsgs.spec.ts for examples

Reply:
Unit test added.